src/headless/utils/init.js

```diff
@@ -248,6 +248,7 @@ function fetchLoginCredentials (wait=0) {
                 }
             };
             xhr.onerror = reject;
+            xhr = await _converse.api.hook('beforeFetchLoginCredentials', this, xhr);
```

Comment:
Please also add a JSDoc string to document this hook. If you look at other hook calls in the code you'll see examples.

Reply:
Done